src/patterns/gen-ai/aws-text-to-sql/README.md

```diff
@@ -0,0 +1,319 @@
+| existingKnowledgeBaseId          | string                                                                                                               | ![Optional](https://img.shields.io/badge/optional-4169E1) | Existing Knowledge base ID.                                                                                                                          |
+
+## Pattern Properties
+
+
+| **Name**            | **Type**                                                                                                        | **Description**                                                  |
+| :-------------------- | :---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
+| vpc                 | [ec2.IVpc](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.Vpc.html)                            | Returns the instance of ec2.IVpc used by the construct           |
+| subnetGroup         | [rds.SubnetGroup](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.SubnetGroup.html)             | Returns the instance of subnet group used by the construct       |
+| lambdaSecurityGroup | [ec2.ISecurityGroup](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.SecurityGroup.html)        | Returns the instance of ec2.ISecurityGroup used by the construct |
+| dbSecurityGroup     | [ec2.ISecurityGroup](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.SecurityGroup.html)        | Returns the instance of ec2.ISecurityGroup used by the construct |
+| configAssetBucket   | [s3.IBucket](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.Bucket.html)                        | Returns the instance of s3.IBucket used by the construct         |
+| databaseCluster     | [rds.DatabaseCluster](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.DatabaseCluster.html)     | Returns the instance of aurora cluster  used by the construct    |
+| dbInstance          | [rds.DatabaseInstance](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.DatabaseInstance.html)   | Returns the RDS db instance  used by the construct               |
+| secret              | [secretsmanager.Secret](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_secretsmanager.Secret.html) | Returns the instance of secret manager used by the construct     |
+
+## Default properties
+
+Out-of-the-box implementation of the construct without any override will set the following defaults:
+
+### VPC, Private Subnet and Security group
+
+- Sets up VPC to deploy the contruct
+  - Private subnet for Amazon Aurora/Amazon RDS
+  - Security group for private subnet
+
+### Amazon Aurora cluster / Amazon RDS instance
```

Comment:
worth mentioning the credentials created (Secrets Manager secret)

Reply:
Moved DB out of the construct. The construct expect a secret with db credentials. - 
- [AWS Secrets Manager Pricing](https://aws.amazon.com/secrets-manager/pricing/)